src/OrderLinesList/Details/OrderLineDetails.test.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import { MemoryRouter } from 'react-router';
+import { Tags } from '@folio/stripes-acq-components';
+
+import OrderLineDetails from './OrderLineDetails';
+import { POLineView } from '../../components/POLine';
+
+import { orderLine, order } from '../../../test/jest/fixtures';
+import { match, history, location } from '../../../test/jest/routerMocks';
+
+jest.mock('@folio/stripes-acq-components', () => ({
+  ...jest.requireActual('@folio/stripes-acq-components'),
+  Tags: jest.fn().mockReturnValue('Tags'),
+}));
+jest.mock('../../components/POLine/POLineView', () => jest.fn().mockReturnValue('POLineView'));
+
+const mutator = {
+  orderLine: {
+    GET: jest.fn().mockResolvedValue(orderLine),
+    PUT: jest.fn(),
+    DELETE: jest.fn().mockResolvedValue(),
+  },
+  order: {
+    GET: jest.fn().mockResolvedValue(order),
+  },
+  locations: {
+    GET: jest.fn(),
+  },
+  materialTypes: {
+    GET: jest.fn(),
+  },
+  funds: {
+    GET: jest.fn(),
+  },
+};
+
+const defaultProps = {
+  refreshList: jest.fn(),
+  resources: {},
+  match: {
+    ...match,
+    params: {
+      id: orderLine.id,
+    },
+  },
+  location,
+  history,
+  mutator,
+};
+
+const renderOrderLineDetails = (props = {}) => render(
+  <OrderLineDetails
+    {...defaultProps}
+    {...props}
+  />,
+  { wrapper: MemoryRouter },
+);
+
+describe('OrderLineDetails', () => {
+  it('should render POLineView', async () => {
+    renderOrderLineDetails();
+
+    const component = await screen.findByText(/POLineView/i);
+
+    expect(component).toBeInTheDocument();
+  });
+});
+
+describe('OrderLineDetails actions', () => {
+  beforeEach(() => {
+    POLineView.mockClear();
+  });
+
+  it('should navigate to order details view', async () => {
+    await waitFor(() => renderOrderLineDetails());
```

Comment:
IMHO it's not idiomatic to await for render

Reply:
changed